.gitconfig

```diff
@@ -0,0 +1,6 @@
+[core]
+	autocrlf = input
+[diff]
+	tool = vimdiff
+[difftool]
+	prompt = false
```

Comment:
are these needed? I have concerns they're portable cross platform

Reply:
Humm yeah that is a good point! I'll remove the tool and leave the prompt so that the confirmation doesn't appear